pages/components/IconLabel.tsx

```diff
@@ -0,0 +1,19 @@
+// IconLabel form fields used for headers
+
+import React, {Component} from "react";
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEnvelope,faSchool,faCheckCircle,faLock,faBoltLightning,faClock,faCalendar,faClockRotateLeft,faIdBadge,faRoad } from "@fortawesome/free-solid-svg-icons";
```

Comment:
Nice use of Font Awesome! 👍

TIP: You can turn on "[Format when save](https://code.visualstudio.com/updates/v1_6#_format-on-save)" feature of Visual Studio Code so the code always nicely formatted.